protos/param/param.proto

```diff
@@ -68,6 +73,27 @@ message SetParamFloatResponse {
     ParamResult param_result = 1;
 }
 
+message ParamListRequest {}
+
+message ParamListResponse {
+    AllParams params= 1;
+}
+
+message IntParam {
+    string name = 1;
+    int32 value = 2;
+}
+
+message FloatParam {
+    string name = 1;
+    float value = 2;
+}
+
+message AllParams {
+    repeated IntParam int_params = 1;
+    repeated FloatParam float_params = 2;
+}
```

Comment:
@JonasVautherin should we consider protobuf maps for this?

https://developers.google.com/protocol-buffers/docs/proto#maps

Reply:
Would be cool, but `protoc-gen-mavsdk` does not understand maps yet. So somebody would have to contribute that, and then the corresponding templates in all the languages. But that would be great, yes!